File: Module3/Module3a/challenge/challenge9.js
// Return the number of vowels in a string
const vowels = ['a', 'A', 'E', 'e', 'I', 'i', 'O', 'o', 'U', 'u']

function count(string) {
    let noOfVowels = 0

    for (let i = 0; i < vowels.length; i++) {
        if (string.includes(vowels[i])) {
            noOfVowels ++
        } 
        
    }
    return noOfVowels
}

console.log(count("Ifeoluwa"))

File: Module4/Module4C/Challenge/challenge9.py
vowels = ['a', 'A', 'E', 'e', 'I', 'i', 'O', 'o', 'U', 'u']

def count(str):
    no = 0
    for i in str:
        if i in vowels:
            no += 1
    return no


print(count("Ifeoluwakiishi"))


# Return the number of vowels in a string
